integrationAPI.01/official/miniPlatform.js

```diff
@@ -513,6 +513,8 @@ $(document).ready(function() {
          }
          if (!taskOptions.hideTitle) {
             $("#task h1").show();
+            if ($("#task h1").length)
+                document.title = "QuickPi - " + $("#task h1").text()
```

Comment:
miniPlatform (and lots of other files) aren't specific to QuickPi at all. We cannot put "QuickPi" in the title there.

Easiest would be to not put "QuickPi" in the title rather than make a complicated handling as anyway this document.title is only displayed to task writers (users will see a title handled by different platforms).

Reply:
Thank you for your remark, in the next commit, the title only correspond to the title of the exercice without any "QuickPi" added.